src/axom/sidre/core/Group.hpp

```diff
@@ -543,6 +545,32 @@ class Group
                     TypeID type,
                     int ndims,
                     IndexType* shape,
+                    Buffer* buff )
+  {
+    return createView(path, type, ndims, shape, nullptr, buff);
+  }
+
+  /*!
+   * \brief Create View object with given name or path in this Group that
+   * has a data description with data type and shape and attach given
+   * Buffer to it.
+   *
+   * If given data type is undefined, or given number of dimensions is < 0,
+   * or given shape ptr is null, method is a no-op.
+   *
+   * This method is equivalent to:
+   * group->createView(name, type, ndims, shape)->attachBuffer(buff), or
```

Comment:
I think this comment needs to be updated to include a permutation parameter in the "equivalent" code